test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -775,14 +761,20 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
 
     It "Validate Invoke-WebRequest returns HTTP errors in exception" {
 
-        $command = "Invoke-WebRequest -Uri http://httpbin.org/status/418"
+        $query = @{
+            body = "I am a teapot!!!"
+            statuscode = 418
+            responsephrase = "I am a teapot"
+        }
+        $uri =  Get-WebListenerUrl -Test 'Response' -Query $query
+        $command = "Invoke-WebRequest -Uri '$uri'"
         $result = ExecuteWebCommand -command $command
 
-        $result.Error.ErrorDetails.Message | Should Match "\-=\[ teapot \]"
+        $result.Error.ErrorDetails.Message | Should be $query.body
         $result.Error.Exception | Should BeOfType Microsoft.PowerShell.Commands.HttpResponseException
```

Comment:
Minor comment - could you please put the type in quotes?

Reply:
fixed